tests/functional/ctlplane/base_test.go

```diff
@@ -614,7 +619,9 @@ func GetDefaultOpenStackControlPlaneSpec() map[string]interface{} {
 		"barbican": map[string]interface{}{
 			"enabled": false,
 		},
-		"openstackclient": map[string]interface{}{},
+		// "openstackclient": map[string]interface{}{
+		// 	"enabled": true,
```

Comment:
the openstackclient doesn't have an enabled parameter. I think we should keep what we had?